src/client/sandbox/node/window.js

```diff
@@ -326,6 +326,15 @@ export default class WindowSandbox extends SandboxBase {
             };
         }
 
+        if (window.DOMParser) {
+            window.DOMParser.prototype.parseFromString = function (...args) {
+                if (args.length > 1 && typeof args[0] === 'string')
```

Comment:
`args.length` is an uncessary check. Drop this.

Also, as I understand we should proxy a string only for the ` text/html` type.

For another supported types:
    "text/xml",
    "application/xml",
    "application/xhtml+xml",
    "image/svg+xml"
 we should not process string.

We have an issue for XHTML  support - https://github.com/DevExpress/testcafe-hammerhead/issues/350

@churkin